packaging/debian/preinst

```diff
@@ -13,9 +14,28 @@ set -e
 # for details, see https://www.debian.org/doc/debian-policy/ or
 # the debian-policy package
 
+supported_archs=(
+    amd64
+)
+
+is_supported_architecture() {
+    for supported_arch in ${supported_archs[@]}; do
+        if [ "$1" == "$supported_arch" ]; then
```

Comment:
`==` with the `[` builtin is a bashism. It won't work when `/bin/sh` is a vanilla POSIX sh, like `dash`. Just use `=`.